ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_control_flow_operators.py

```diff
@@ -203,3 +204,50 @@ def _test_body_fn(x):
         body_fun=_test_body_fn,
         init_val=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.lax.scan",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_value=-1000,
+        max_value=1000,
+        min_num_dims=1,
+        min_dim_size=1,
+    ),
+)
+def test_jax_lax_scan(
+    *,
+    dtype_and_x,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    def _test_f(carry, x):
+        y = carry * x
+        new_carry = x + y
+        return new_carry, y
+
+    input_dtype, x = dtype_and_x
+    length = len(x)
+    expected_carry = 0
+    expected_ys = []
+    carry = expected_carry
+    for elem in x:
+        carry, y = _test_f(carry, elem)
+        expected_ys.append(y)
+    expected_result = (expected_carry, ivy.stack(expected_ys))
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        f=_test_f,
+        # init=0,
+        xs=x,
+        length=length,
+        expected_result=expected_result,
```

Comment:
All function args need to be passed here as kwargs, including 'init' and the ones added in our modified function signature.